src/plugins/discover/public/application/components/table/table_cell_actions.tsx

```diff
@@ -47,12 +45,12 @@ export const TableActions = ({
       />
       <DocViewTableRowBtnToggleColumn
         active={isActive}
-        fieldname={fieldName}
-        onClick={toggleColumn}
+        fieldname={String(fieldKey)}
```

Comment:
fieldKey should always be string, right?

Reply:
Yes, it should be a string, thanks